### ISSUE TITLE
[Android] "Night Mode" menu accessibility & page-specific settings

### ISSUE THREAD

**Issue by Acumane** [brave/brave-browser#21239]
## Description
Setting for toggling "Night Mode" shouldn't be buried so deep in settings. Additionally, It'd be nice if the setting preference was remembered on a per-page basis.

## Steps to reproduce 

   1. Tap `⋮` 
   2. Tap `Settings`
   3. Scroll down to `Display`
   4. Tap `Appearance`
   5. Toggle `Enable "Night Mode" (Experimental)`

## Actual result 
Unreasonably lengthy action required to enable/disable a commonly used setting

## Expected result
Toggleable from either the bottom bar or hamburger menu (`⋮`)

## Issue reproduces how often 
Perpetual

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? **Y**
- Can you reproduce this issue with the current Play Store Beta version? **--**
- Can you reproduce this issue with the current Play Store Nightly version? **--**

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: Android 12
   
## Brave version
- Brave 1.35.103
- Chromium 98.0.4758.102

**Comment by JacenBridger** [comment]
+1

**Comment by JacenBridger** [comment]
Cc @srirambv @SergeyZhukovsky

**Comment by SergeyZhukovsky** [comment]
Cc @anthonypkeane @timchilds for insights

**Comment by SergeyZhukovsky** [comment]
yeah, I agree, it would be cool to make it per site as well as some websites look bad with it, most are ok. Some exceptions would be cool as @JacenBridger mentioned.
